akeneo-design-system/.prettierrc

```
@@ -0,0 +1,13 @@
+{
+    "arrowParens": "avoid",
+    "bracketSpacing": false,
+    "jsxBracketSameLine": false,
+    "jsxSingleQuote": true,
+    "printWidth": 120,
+    "quoteProps": "as-needed",
+    "semi": true,
+    "singleQuote": true,
+    "tabWidth": 4,
```

```suggestion
    "tabWidth": 2,
```
As in https://github.com/akeneo/pim-community-dev/blob/master/.prettierrc.json

---

We have to decide about this ;) 
https://github.com/akeneo/pim-community-dev/blob/master/src/Akeneo/Connectivity/Connection/front/.prettierrc.json#L10
https://github.com/akeneo/pim-community-dev/blob/master/src/Akeneo/Connectivity/Connection/front/.editorconfig#L6
make one file to rule them all, if everyone is ok.

---

AH! I wasn't even aware that they had a custom prettier config in `Connectivity/Connection` ;)